### PR TITLE
Fix typos, improve wording in a number of different places

### PR DESCRIPTION
**Pull request by jhoak**

### CODE REVIEW

File: organizational/linkstate.rst [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 
 **Chair of Speakers:** This person will handle the logistics and scheduling of
 talks.  They will *not* be solely responsible for soliciting speakers -- this is
-a duty that all HacSoc members should hare.
+a duty that all HacSoc members should share.
 
 - They will hold the whip on getting members to invite speakers.
 - They will evaluate talk proposals.
-- They are the person who will finalize the schedule: determine time slots, do
+- Ultimately, they will finalize the schedule by determining time slots, doing
```
Comment (mtbentley) [inline organizational/linkstate.rst]: `doing` sounds awkward

**Comment by mtbentley** [comment]
Thanks for these changes!  I'll leave it open for other people to comment and for changes to be made, but this is a much appreciated update.

File: organizational/linkstate.rst (hunk 1 above)
Comment (jhoak) [reply organizational/linkstate.rst]: "handling the logistics" then? (edit to lower bloat) or managing?

Comment (jhoak) [inline organizational/linkstate.rst]: managing?

Comment (mtbentley) [reply organizational/linkstate.rst]: handling or managing would work now that you mention it

**Comment by jhoak** [comment]
Updated

**Review by mtbentley — APPROVED** [review]
lg2m.  I'll give it a day for other comments before merging

**Comment by jhoak** [comment]
Scanned thru the changes once more and found I needed to add length to a couple headers -- that's done in the latest commit (sorry for the spam)

**Review by dfan14051 — COMMENTED** [review]
Proofread the final round of changes. Looks good to me

File: organizational/slack.rst [hunk 2]
```diff
@@ -1,12 +1,12 @@
 HacSoc Slack
 ============
 
-`Hacsoc Slack`_ 
+`Click here to go to the official HacSoc Slack.`_ 
 
 .. _Hacsoc Slack: https://hacsoc.slack.com
```
Comment (brenns10) [inline organizational/slack.rst]: This change will almost certainly break the link. Notice how the reference on line 6 matches the link text on line 4. You can either make them match again, or switch to the inline link syntax:

    `link text <link target>`_

[Quick reference](http://docutils.sourceforge.net/docs/user/rst/quickref.html#hyperlink-targets). Be sure to check these things with a `make html` - there will be "compiler warnings" for broken links, etc.

**Review by brenns10 — CHANGES_REQUESTED** [review]
All the content looks fine, but there appears to be a broken link. While you're at it, make sure `make html` doesn't give any more errors.

File: organizational/slack.rst (hunk 2 above)
Comment (mtbentley) [reply organizational/slack.rst]: Hmm it seems that we should add Travis to check this automagically?

Comment (brenns10) [reply organizational/slack.rst]: @matthewbentley !! *do not* enable Travis yet

Comment (brenns10) [reply organizational/slack.rst]: If you enable it for pull requests, then it will update the site

Comment (brenns10) [reply organizational/slack.rst]: I'll need to update `push.sh` to check whether it is within a PR (just an environment variable check, NBD), but we can't switch it yet

**Review by brenns10 — APPROVED** [review]

**Comment by jhoak** [comment]
Subsections broke again (a git mistake on my end removed the extra lines I added in an earlier commit) so I fixed those again. Everything should be 👌 now

File: organizational/slack.rst (hunk 2 above)
Comment (mtbentley) [reply organizational/slack.rst]: Yeah I won't change anything yet.  Just a suggestion for the future.

**Pull request merged**